src/common/options.cc

```diff
@@ -2479,6 +2479,14 @@ std::vector<Option> get_global_options() {
     .set_default(24)
     .set_description(""),
 
+    Option("osd_scrub_begin_week_day", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("the begin week day permit to scub,0 sunday,1 monday and so forth"),
```

Comment:
'permit to scub'?

Reply:
any confusion? the relation is [begin,end)